patroni/dcs/etcd3.py

```diff
@@ -725,10 +771,11 @@ def _load_cluster(self, path, loader):
         except Exception as e:
             self._handle_exception(e, 'get_cluster', raise_ex=Etcd3Error('Etcd is not responding properly'))
         self._has_failed = False
+        assert cluster is not None
         return cluster
 
     @catch_etcd_errors
-    def touch_member(self, data):
+    def touch_member(self, data: Dict[str, Any]) -> bool:
```

Comment:
```suggestion
    def touch_member(self, data: Dict[str, Any]) -> Union[Dict[str, Any], bool]:
```
Due to the return type of put method on line 792

Reply:
No, it has to be boolean. I'll just add an explicit cast.